flusher.go

```diff
@@ -60,7 +62,12 @@ func (s *Server) Flush(ctx context.Context) {
 	span.Add(s.computeMetricsFlushCounts(ms)...)
 
 	if s.IsLocal() {
-		go s.flushForward(span.Attach(ctx), tempMetrics)
+		// Forward over gRPC or HTTP depending on the configuration
```

Comment:
Maybe not?  Theoretically both of the downstream Veneurs could submit the same aggregated values to the sinks.  I think you'd need to at a minimum, make sure of the following:
* The flush intervals on the upstream Veneur(s) are aligned
* All local veneurs sending the same metrics forward over both HTTP *and* gRPC
* Your HTTP and gRPC forwards go to different upstream Veneur(s).  Otherwise I think you'd effectively forward each metric twice to the same set of Veneurs!

It feels like it might be tricky to get right.  Any particular reason you you're considering this?

Reply:
It's an outgrowth of the question about the config - whether we could just fully separate the options for HTTP and gRPC, and if that would then imply allowing both of them to be run simultaneously. Your answers here help clarify the implications of that (i'm still wrapping my head around the behavior here in general) - it makes sense that while we might want to accept using either protocol, we only want to _send_ over one.

Reply:
Got it.  I think your description sounds right, accepting over HTTP and gRPC should be fine as long as nothing is *submitting* over both.  The changes that separate out the `forward_address` and `grpc_forward_address` options now make initialization throw an error if they're both set, which should stop this from happening.